src/base/bittorrent/session.cpp

```diff
@@ -127,6 +127,49 @@ namespace
         return result;
     }
 
+    QStringList setToList(const QSet<QString> &set)
```

Comment:
I appreciate your patience, and apologize to the reviewers for not catching more of these minor style issues before sending the PR. I went through the diffs again and tried to apply all the general comments. Sorry if I have missed anything.